resources/dicts/thoughts/alive/apprentice.json

```
@@ -34,7 +34,20 @@
             "Hopes to have an easy training session",
             "Is relaxing in between training and patrols",
             "Has so much to learn",
-            "Wants to learn something new today"
+            "Wants to learn something new today",
+            "Is gossiping",
+            "Is acting angsty",
+            "Is relaxing in between training and patrols",
+            "Hopes to have an easy training session",
+            "Is dreading {PRONOUN/m_c/poss} apprentice duties",
+            "Turns out, prey is much harder to hunt then bugs",
```

not the correct phrasing for a thought, could try this instead
```suggestion
            "Discovers that prey is much harder to hunt than bugs",
```